### DISCUSSION TITLE
github student developer pack

### DISCUSSION THREAD

**Discussion by navjeet101709** [community/community#18365] [GitHub Education]
No option available for me to for Uploading proof of your academic status.
Sir I want to upload by schoold ID card instead of taking picture because my laptop camera is not of good quality to click picture
![No Option for upload file from computer](https://user-images.githubusercontent.com/107310686/173194596-28c828af-202c-4ea1-8522-a709c9b19b25.png)
.

**Comment by Sulagna-Dutta-Roy** [comment]
You can upload PDF file also instead of clicking file
